db/migrate/20150318015754_add_auto_logout_minutes_to_relays.rb

```diff
@@ -0,0 +1,5 @@
+class AddAutoLogoutMinutesToRelays < ActiveRecord::Migration
+  def change
+    add_column :relays, :auto_logout_minutes, :integer, null: false, default: 60
```

Comment:
It seems like this should be nullable for relays that don't have auto_logout. It'll seem weird to see that be true and this be 60 in the DB.